src/web-ui/src/public/ProductDetail.vue

```diff
@@ -214,6 +215,22 @@ export default {
         let root_url = process.env.VUE_APP_IMAGE_ROOT_URL
         return root_url + this.product.category + '/' + this.product.image
       }
+    },
+    productStock: function () {
+      if (this.product.currentstock > 0 ) {
+        return "Items currently in stock : " + this.product.currentstock 
+      }
+      else {
+        return "Sorry this item is currently out of stock"
+      }
```

Comment:
The template is a better place for messages rather than in code. Keeps the presentation elements together. You can use Vue [conditional rendering](https://vuejs.org/v2/guide/conditional.html) to display different messages based on the stock level. So something like this:

```html
<span v-if="product.current_stock > 0">Items currently in stock: {{ product.current_stock }}</span>
<span v-else>Sorry, this item is currently out of stock</span>
```